compute/instances/api-cli/snapshot-import-export-feature.mdx

```diff
@@ -0,0 +1,174 @@
+content:
+  h1: Using snapshot import/export features with the API
+  paragraph: This page explains how to import/export Instances snapshots using the Scaleway API
+dates:
+  validation: 2022-07-25
+  posted: 2021-07-25
+---
+
+The snapshot transfer feature allows to export or import Instances snapshots. 
+Snapshots are created using the QCOW2 format and stored in a Scaleway Objetct Storage Bucket that holds the data. These images can then be used to create a copy of an Instance in a different AZ.
+
+More information on this file format, and how to use it can be found in the [QEMU documentation](https://www.qemu.org/docs/master/system/images.html#cmdoption-image-formats-arg-qcow2).
+
+<Message type="tip">
+
+This document shows you how to use this functionality using the Scaleway API. The import/export feature is also available in the Console, CLI, and SDK.
+
+</Message>
+
+<Message type="requirement">
+
+  - You have a [Scaleway account](/Console/my-account/how-to/create-an-account/)
+  - You have [configured your API keys](/Console/my-project/how-to/generate-api-key/)
+  - You have an Instance using [Block Storage volumes](/storage/block/how-to/create-a-volume)
+  - You have created a [snapshot](/compute/instances/how-to/create-a-snapshot)
+  - You have created an [Object Storage bucket](/storage/object/how-to/create-a-bucket)
+
+</Message>
+
+## Exporting snapshots
+
+1. Create a Scaleway Object Storage bucket.
+    You need an S3 bucket to export your QCOW2 file into. Any bucket that belongs to the same project as the snapshot can be used. However, if you don't have one already, you can [create it](https://www.scaleway.com/en/docs/storage/object/how-to/create-a-bucket/) in the Console.
+
+2. Create a snapshot from a volume.
+    You will need to create a snapshot from the volume you want to export to use this functionality. If you have not done so already, refer to the [documentation](https://www.scaleway.com/en/docs/compute/instances/how-to/create-a-snapshot#how-to-create-a-snapshot) for instructions on how to create a snapshot.
+
+3. Call the `export` snapshot API endpoint to initiate the snapshot export. 
+    For example, using curl:
+    ```json
+    $ curl -X POST https://api.scaleway.com/instance/v1/zones/fr-par-1/snapshots/<snapshot_uuid>/export -H "X-Auth-Token: <scaleway_secret_id>" -H "Content-Type: application/json"  --data '{"bucket": "my-bucket", "key": "my-exported-snapshot.qcow2"}'
+    ```
+    The following information is required to run the command above: 
+     - The Availability Zone of the snapshot (here: `fr-par-1`)
+     - The secret key of your API keypair (`<scaleway_secret_id>`)
+     - The snapshot ID (`<snapshot_uuid>`)
+     - The name of the Object Storage bucket to store the snapshot (which has to exist in the same Scaleway region as the snapshot)
+     - A key (can be any acceptable key for Scaleway S3 ( suffixing qcow2 images with .qcow2)
```

Comment:
the key part isn't very clear for me - is it the name for the snapshot?

Reply:
Yes it is the file name of the object